### PR TITLE
Refactored some classes

### PR DESCRIPTION
**Pull request by odilonjk**

### CODE REVIEW

File: src/main/java/br/com/tareffa/fileupload/controllers/UserController.java [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
+import br.com.tareffa.fileupload.services.UserService;
 
 @RestController
-@RequestMapping(path = {"/users"})
+@RequestMapping("/users")
 public class UserController {
 
-    @Autowired
-    UserRepository userRepository;
+	@Inject
```
Comment (odilonjk) [inline src/main/java/br/com/tareffa/fileupload/controllers/UserController.java:25]: I've changed from `@Autowired` to `@Inject` to follow the standard dependency injection.

File: src/main/java/br/com/tareffa/fileupload/controllers/UserController.java [hunk 2]
```diff
@@ -1,34 +1,48 @@
...
+	UserService userService;
 
-    @GetMapping(path = {""})
-    public ResponseEntity<List<User>> findUsers(
-            @RequestParam("page_size") Integer pageSize,
-            @RequestParam("page_index") Integer pageIndex) throws Exception {
-        return ResponseEntity.ok(userRepository.findAll(pageSize, pageIndex));
-    }
+	@GetMapping()
+	public ResponseEntity<List<User>> findUsers(@RequestParam(defaultValue = "10", required = false) Integer pageSize,
```
Comment (odilonjk) [inline src/main/java/br/com/tareffa/fileupload/controllers/UserController.java:29]: I've put some default values here, just as a good practice.

File: src/main/java/br/com/tareffa/fileupload/controllers/UserController.java [hunk 3]
```diff
@@ -1,34 +1,48 @@
...
+			@RequestParam(defaultValue = "0", required = false) Integer pageIndex) throws Exception {
+		try {
+			return ResponseEntity.ok(userService.findAll(pageSize, pageIndex));
```
Comment (odilonjk) [inline src/main/java/br/com/tareffa/fileupload/controllers/UserController.java:32]: As a good practice, classes at this layer should never know repository classes.

**Review by mtslucasmartins — APPROVED** [review]
Thanks for the feedback!

**Pull request merged**